src/runtime/vulkan_resources.h

```diff
@@ -1625,24 +1632,45 @@ void vk_destroy_compiled_shader_module(VulkanCompiledShaderModule *shader_module
     }
 
     if (shader_module->shader_bindings) {
+#ifdef DEBUG_RUNTIME
+        debug(user_context)
+            << "  destroying shader bindings ("
+            << "shader_module: " << shader_module << ", "
+            << "shader_bindings: " << shader_module->shader_bindings << ")\n";
+#endif
         for (uint32_t n = 0; n < shader_module->shader_count; n++) {
+#ifdef DEBUG_RUNTIME
+            debug(user_context) << "  destroying shader binding [" << n << "] ";
+            if (shader_module->shader_bindings[n].entry_point_name) {
+                debug(user_context) << shader_module->shader_bindings[n].entry_point_name << "\n";
+                vk_host_free(user_context, shader_module->shader_bindings[n].entry_point_name, allocator->callbacks());
```

Comment:
This string is only freed when the runtime in compiled in debug mode. Yet the string is always allocated (regardless of debug feature flag).

Reply:
Oops, thanks!  Yes, that should not have a DEBUG_RUNTIME guard around it.  Fixed.